src/qilib/configuration_helper/adapters/d5a_instrument_adapter.py

```diff
@@ -46,11 +49,34 @@ def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
             raise SpanValueError('D5a instrument has span unequal to "4v bi"')
 
     def apply(self, config: PythonJsonStructure) -> None:
-        """ Step values for dacs should be part of configuration."""
-        super().apply(config)
+        """ Does not apply config, except for instrument name, to device, but compares config to device settings.
```

Comment:
This is not true, it does apply settings of the `step` and `inter_delay`

Reply:
Updated.